financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -536,6 +548,28 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
         setState { copy(viewEffect = FinishWithResult(result, finishMessage)) }
     }
 
+    /**
+     * On scenarios where native failed mid flow due to attestation errors, switch back to web flow.
+     */
+    private fun switchToWebFlow() {
+        integrityVerdictManager.setVerdictFailed()
```

Comment:
This side-effect seems like it doesn’t belong in this method.

Reply:
moved it out!